src/AmbientSounds.Uwp/Services/DialogService.cs

```diff
@@ -90,7 +90,7 @@ public async Task<IList<string>> OpenShareResultsAsync(IList<string> soundIds)
             IsDialogOpen = false;
 
             return result == ContentDialogResult.Primary
-                ? content.ViewModel.Sounds.Where(x => x.IsInstalled).Select(x => x.Id).ToList()
+                ? content.ViewModel.Sounds.Where(static x => x.IsInstalled).Select(static x => x.Id).ToList()
```

Comment:
This is new to me. Do you have docs that I can read up about this? Does it lead to a performance gain?

Reply:
Those are [static anonymous functions](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/static-anonymous-functions), introduced in C# 9. They're purely syntactic sugar, so there's no functional difference at all between not using `static` here (nor there's any necessary runtime support). What they're doing is just to prevent you from capturing any variable in the lambda, effectively guaranteeing that no closure will be generated. 

You can still achieve the same without `static` and by just being careful, but this gives you:
- More explicitness, as when glancing at the code you immediately tell which lambdas are stateful and which ones aren't
- It gives you more safety if you accidentally made changes and introduced a capture, as the code would fail to build

Personally I'd recommend to always add `static` to lambdas by default where applicable for these two reasons 🙂